arcade/examples/depth_of_field.py

```diff
@@ -171,7 +178,7 @@ def render(self):
         self._geo.render(self._render_program)
 
 
-class App(Window):
+class App(View):
```

Comment:
convention is `GameView`. Even though it's not really accurate, the examples used to be a mess in this regard, so im forcing a naming convention.